Sources/MVTTools/VectorTileDecoder.swift

```diff
@@ -121,7 +121,7 @@ extension VectorTile {
                 return Double(value.floatValue)
             }
             else if value.hasUintValue {
-                return UInt(value.uintValue)
+                return UInt64(value.uintValue)
             }
             else if value.hasSintValue {
                 return Int(value.sintValue)
```

Comment:
@joluet What about this `Int` here?

Reply:
Good point. That should be `Int64` too. I've added that change now. I tested this on an Apple watch.